src/locales/vocabulary.js

```diff
@@ -99,47 +100,42 @@ export default {
   'editdialog.anonymousMessage2_login': 'log in to OpenStreetMap',
   'editdialog.anonymousMessage3': ', your changes will be immediate.',
   'editdialog.add_major_tag': 'Add',
-  'editdialog.location_checkbox': 'Change location',
+  'editdialog.location_checkbox': 'New location',
   'editdialog.location_placeholder': 'eg. across the street',
   'editdialog.location_editor_to_be_added': 'The position cannot be edited here yet, you can do so in the <a href="__link__">iD editor</a>.',
-  'editdialog.place_cancelled': 'Place permanently closed (delete)',
+  'editdialog.place_cancelled': 'Permanently closed (delete)',
   'editdialog.comment': 'Comment (optional)',
-  'editdialog.comment_placeholder': 'link to the source of information etc.',
+  'editdialog.comment_placeholder': 'Extra details, link to source, etc.',
   'editdialog.info_edit': `Your edit will be immediately saved to the OpenStreetMap. Please,
          enter only information from your own or verified sources. It is prohibited
          to copy copyrighted data (e.g. Google Maps). <a href="https://wiki.openstreetmap.org/wiki/How_We_Map">More info</a>`,
-  'editdialog.info_note': `Your suggestion will be processed by OpenStreetMap volunteers. Here
-         you can add an additional note or describe position adjustment etc.
-         It is also appropriate to support your contribution with a link to a source of information (web,
-         photo, etc.).`,
-  'editdialog.other_tags': 'Other properties – tags',
+  'editdialog.info_note': `Your suggestion will be reviewed by editors. You can add additional information for them below! If possible, please provide a link to the source that your suggestion is based on.`,
+  'editdialog.other_tags': 'Edit Tags',
   'editdialog.other_tags.new_key': 'new key',
-  'editdialog.other_tags.add': 'Add another tag',
+  'editdialog.other_tags.add': 'Add tag',
   'editdialog.other_tags.will_be_deleted': 'will be deleted',
-  'editdialog.other_tags.info': `The Tags describe properties
-        of each map feature in agreed format. Here you will find the complete
-        <a href="https://wiki.openstreetmap.org/wiki/Map_Features">overview of all tags in OpenStreetMap</a>.`,
+  'editdialog.other_tags.info': `Tags contain the data used to display objects on the map!<br>You can find <a href="https://wiki.openstreetmap.org/wiki/Map_Features">a reference for all tags on the OpenStreetMap Wiki</a>!`,
 
-  'editsuccess.close_button': 'Close',
+  'editsuccess.close_button': 'Done',
   'editsuccess.note.heading': 'Thank you for your suggestion!',
-  'editsuccess.note.subheading': 'OpenStreetMap volunteers will process it over time.',
-  'editsuccess.note.body': 'It usually takes few days. However, in places with no active community, it can take a very long time.',
+  'editsuccess.note.subheading': 'OpenStreetMap volunteers will review it.',
+  'editsuccess.note.body': "This usually takes few days but could take longer if it's not a high traffic area.",
```

Comment:
Again, i would like to mention comunity. Still the point of this app is to educate the users what processes lies behind the interface. Eg. so they are not angry that it takes so long (while in some comerical it is in two days).

Reply:
The issue I had was that the "active community" it was referring to was ambiguous. It sounded as if the change would not be made quickly if the actual community living in that location was sedentary :joy:

Maybe:
> It usually takes a few days for a change to be reviewed. If there are no volunteers working on the area it could take longer!

Reply:
Yes, your suggestions sounds good to me now 🙂 The truth is, that some notes can take several years to process. There isnt really any process around it, and its up to random individuals to check new notes once in a while and do something about it 😁 

Btw, have you editted the map in iD editor? I'd say its pretty delightful, and in the end you recieve specific links to local communities depending on the place of edit. Something we could someday as well 🙂 This is the [index](https://github.com/osmlab/osm-community-index).

Reply:
I'll do 
> It usually takes a few days for a change to be reviewed. If there are no volunteers working on the area it could take a while!

And yes, the iD editor has been my only editing experience so far actually, could be cool to adopt the communities feature.